test/shared/src/main/scala/zio/test/MutableRunnableSpec.scala

```diff
@@ -39,25 +40,27 @@ import scala.util.control.NoStackTrace
  * }
  * }}}
  */
-class MutableRunnableSpec[R <: Has[_]](
+class MutableRunnableSpec[R <: Has[_]: Tag](
   layer: ZLayer[TestEnvironment, Throwable, R],
-  aspect: TestAspect[R, R, Any, Any]
+  aspect: TestAspect[R with TestEnvironment, R with TestEnvironment, Any, Any] = TestAspect.identity
 ) extends RunnableSpec[TestEnvironment, Any] {
   self =>
 
+  type RT = R with TestEnvironment
```

Comment:
Can we just use `R with TestEnvironment` instead of creating this new type alias?

Reply:
Removed